kymatio/scattering3d/frontend/torch_frontend.py

```diff
@@ -118,20 +118,23 @@ def forward(self, input_array):
                 'The input should be a torch.cuda.FloatTensor, '
                 'a torch.FloatTensor or a torch.DoubleTensor.')
 
-        input_array = input_array.contiguous()
+        if input_array.dim() < 3:
+            raise RuntimeError('Input tensor must have at least three '
+                               'dimensions.')
 
         if (input_array.shape[-1] != self.O or input_array.shape[-2] != self.N
             or input_array.shape[-3] != self.M):
             raise RuntimeError(
                 'Tensor must be of spatial size (%i, %i, %i).' % (
                     self.M, self.N, self.O))
 
-        if input_array.dim() != 4:
-            raise RuntimeError('Input tensor must be 4D.')
+        input_array = input_array.contiguous()
```

Comment:
Is it possible to rather add a test, than `contiguous`. There will be a common test shared across backends in the PR `simplifications`.

Reply:
Yes but let's make an issue.